konig-workbook-update-maven-plugin/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>konig-parent</artifactId>
+		<groupId>io.konig</groupId>
+		<version>2.0.0-8-SNAPSHOT</version>
+	</parent>
+
+	<groupId>io.konig</groupId>
+	<artifactId>konig-workbook-update-maven-plugin</artifactId>
```

Comment:
I see that you are introducing a new maven module (konig-workbook-update-maven-plugin).  You need to add this module to konig-parent/pom.xml.